.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   lint:
     docker:
-      - image: python:3.7
+      - image: cimg/python:3.10.0
```

Comment:
Granted, the freshest version we use anywhere, but I figure these easy-to-fix repos with low technical complexity are a good place to take new images for a spin.